### PR TITLE
Feature get customer phone lines

### PR DESCRIPTION
**Pull request by parksamuelsoung**
New Fetaure. Get phone lines of each customer! Made logical reference to customer from phone line entity. Endpoint tests passed.
Screenshot (88)
Screenshot (87)

### CODE REVIEW

File: src/main/java/com/skillstorm/spartanwireless/controllers/PhoneLineController.java [hunk 1]
```diff
@@ -19,34 +19,40 @@
...
+    primary key of its own entity. We will consider adding this in the future.
     @PutMapping("/{phoneNumber}")
     public ResponseEntity<PhoneLineResponseDto> updatePhoneLine(@PathVariable String phoneNumber, @RequestBody PhoneLineRequestDto phoneLineRequestDto) {
         return new ResponseEntity<>(phoneLineService.updatePhoneLine(phoneNumber, phoneLineRequestDto), HttpStatus.OK);
     }
+    */
 
     @DeleteMapping("/{phoneNumber}")
-    public void deleteById(@PathVariable String phoneNumber) {
+    public void deleteById  (@PathVariable String phoneNumber) {
```
Comment (sod2003) [inline src/main/java/com/skillstorm/spartanwireless/controllers/PhoneLineController.java:55]: Was this a typo?

File: src/main/java/com/skillstorm/spartanwireless/dtos/PhoneLineResponseDto.java [hunk 2]
```diff
@@ -13,6 +13,4 @@
 public class PhoneLineResponseDto {
     
     private String phoneNumber;
-    private Customer customer;
-    private Device device;
```
Comment (sod2003) [inline src/main/java/com/skillstorm/spartanwireless/dtos/PhoneLineResponseDto.java:17]: Are we worried that removing Device from the Response would limit the ability to display the associated Device with the Customer Phone Line on the Front-End?

**Review by sod2003 — COMMENTED** [review]
I'm looking at this, and it's making me think about how we want to handle the Customer Interface and how they pick the Device they assign to their Phone Line. I think we definitely need to have a conversation, as a Team, about what we will allow the Customer to do from the perspective of the Front-End. Can they assign devices to phone lines if they can't update the phone lines table?

File: src/main/java/com/skillstorm/spartanwireless/controllers/PhoneLineController.java (hunk 1 above)
Comment (parksamuelsoung) [reply src/main/java/com/skillstorm/spartanwireless/controllers/PhoneLineController.java:55]: yes

Comment (parksamuelsoung) [reply src/main/java/com/skillstorm/spartanwireless/controllers/PhoneLineController.java:55]: will fix

File: src/main/java/com/skillstorm/spartanwireless/dtos/PhoneLineResponseDto.java (hunk 2 above)
Comment (parksamuelsoung) [reply src/main/java/com/skillstorm/spartanwireless/dtos/PhoneLineResponseDto.java:17]: This is a circular dependency issue that had to be resolved/

Comment (parksamuelsoung) [reply src/main/java/com/skillstorm/spartanwireless/dtos/PhoneLineResponseDto.java:17]: I will have to work on the device customer phone line dependecies. I might do on next pr though.

Comment (sod2003) [reply src/main/java/com/skillstorm/spartanwireless/dtos/PhoneLineResponseDto.java:17]: Okay, cool.

**Review by sod2003 — APPROVED** [review]

**Pull request closed without merging**